File: Backend/htbp_parent/service/service_user/src/main/java/com/seclib/htbp/user/ServiceUserApplication.java
package com.seclib.htbp.user;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.openfeign.EnableFeignClients;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan(basePackages = "com.seclib.htbp")
@EnableDiscoveryClient
@EnableFeignClients(basePackages = "com.seclib.htbp")
public class ServiceUserApplication {
    public static void main(String[] args) {
        SpringApplication.run(ServiceUserApplication.class, args);
    }
}

File: Backend/htbp_parent/service/service_user/src/main/java/com/seclib/htbp/user/ServiceUserApplication.kt
package com.seclib.htbp.user


import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.cloud.openfeign.EnableFeignClients
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@ComponentScan(basePackages = ["com.seclib.htbp"])
@EnableDiscoveryClient
@EnableFeignClients(basePackages = ["com.seclib.htbp"])
open class  ServiceUserApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(ServiceUserApplication::class.java, *args)
        }
    }
}